README.md

```diff
@@ -70,6 +70,8 @@ For usage information, use the default target:
 make
 ```
 
+If you are developing on Windows, [msys2](https://www.msys2.org/) may be used for the required build tools. Follow all the suggested installation steps for a complete environment.
```

Comment:
@markXLIX if you get a chance, could you try msys2 on this branch (or later, after it's merged) and see if the Makefile targets work for you and possibly suggest documentation updates?

Reply:
Yes, I will try it out today.